build.ps1

```diff
@@ -45,9 +45,9 @@ function DotnetPack {
 
 function ProcessLastExitCode {
     param($exitCode, $msg)
-    if ($lastexitcode -ne 0)
+    if ($exitCode.Equals(0))
```

Comment:
Question: why `$lastexitcode` before,but '$exitCode' now? should we follow the camel naming rule?